pages/agent/clusters.md

```diff
@@ -90,6 +90,23 @@ To add a maintainer to a cluster:
 1. Select a user or team.
 1. Click _Add Maintainer_.
 
+### Pause a queue
+
+You can pause a queue to prevent jobs from being dispatched to agents associated with that queue.
+To pause a queue:
+
+1. Navigate to your cluster’s _Queues_.
+1. Select on the queue you wish to pause.
+1. Select _Edit_.
+1. Under _Queue Management_, select _Pause Queue_.
+1. Enter an optional note in the dialog if needed, and confirm that you wish to pause the queue. The note will be displayed on the queue page, and any affected builds.
```

Comment:
I think the "if needed" might be playing another role. For me, it's hinting at the reason why you might or might not leave a note. Maybe it would be helpful to be more explicit about that? Thinking:

```suggestion
1. Enter an optional note in the dialog, and confirm that you wish to pause the queue. 
   
    You can use the note to explain why you're pausing the queue. The note will display on the queue page and any affected builds.
```

Note, the spacing should display the clarifying sentence indented with the list. I think I've got the spacing right, but worth double checking.